docs/technical-reference/api/moduleTemplate-cr.md

```diff
@@ -36,7 +36,6 @@ spec:
       resources:
         operator:
           limits:
-            cpu: "800m"
```

Comment:
This is related to module CR, we do not care if cpu limit should be set or not. In certain scenario, limit CPU is needed.